recipes/hictk/all/conanfile.py

```diff
@@ -39,20 +40,26 @@ def layout(self):
         cmake_layout(self, src_folder="src")
 
     def requirements(self):
-        self.requires("bshoshany-thread-pool/3.5.0", transitive_headers=True)
-        self.requires("fast_float/6.0.0", transitive_headers=True)
+        self.requires("bshoshany-thread-pool/4.0.1", transitive_headers=True)
+        self.requires("fast_float/6.1.0", transitive_headers=True)
         if self.options.with_eigen:
             self.requires("eigen/3.4.0", transitive_headers=True)
-        self.requires("fmt/10.1.1", transitive_headers=True)
+        self.requires("fmt/10.2.1", transitive_headers=True)
         self.requires("hdf5/1.14.3", transitive_headers=True)
-        self.requires("highfive/2.8.0", transitive_headers=True)
+        self.requires("highfive/2.9.0", transitive_headers=True)
         self.requires("libdeflate/1.19", transitive_headers=True)
-        self.requires("parallel-hashmap/1.37", transitive_headers=True)
+        self.requires("parallel-hashmap/1.3.11", transitive_headers=True)  # Note: v1.3.11 is more recent than v1.37
         self.requires("span-lite/0.10.3", transitive_headers=True)
-        self.requires("spdlog/1.12.0", transitive_headers=True)
-        self.requires("xxhash/0.8.2", transitive_headers=True)
+        self.requires("spdlog/1.13.0", transitive_headers=True)
         self.requires("zstd/1.5.5", transitive_headers=True)
 
+        if Version(self.version) < "0.0.7":
```

Comment:
I still see xxhash listed in 0.0.7. What am I missing? 

- https://github.com/paulsengroup/hictk/blob/v0.0.7/CMakeLists.txt#L123
- https://github.com/paulsengroup/hictk/blob/v0.0.7/cmake/modules/FindxxHash.cmake

Reply:
Good catch! Those are leftovers from back when hictk linked to xxhash.
The dependency was actually removed with v0.0.4 (see https://github.com/paulsengroup/hictk/commit/af52cb861a21eb1206e7332e16da097d3e8c0d50)
I am updating the version check in the conanfile.py just in case someone requests v0.0.4-7 to be added to conancenter.

Reply:
Thank for the reference!